governance/votes/Executive vote - February 22, 2024.md

```diff
@@ -0,0 +1,115 @@
+- Increase the DIRECT-SPARK-DAI [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) by 12 hours from 12 hours to **24 hours**.
+- Increase the DIRECT-SPARK-DAI buffer by 20 million DAI from 30 million DAI to **50 million DAI**.
+
+### Housekeeping Actions
+
+**Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697/2), [2](http://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697/4)\
+**Proposal:** [Forum post "Proposed Housekeeping items (Upcoming Executive Spell - 2024-02-22)"](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-02-22/23697)
+
+#### Push USDP out of the Input Conduit
+
+If this executive proposal passes, following the recommendation of the Governance Facilitators in collaboration with the spell team, the following operations will be performed:
+
+- Raise `PSM-PAX-A` DC to **1,000,000 DAI**
+- Call push() on `MCD_PSM_PAX_A_INPUT_CONDUIT_JAR` (use push(uint256 amt)) to push **754,598.72 USDP**
+- Call void() on `MCD_PSM_PAX_A_JAR`
+- Set `PSM-PAX-A` DC to **0 DAI**
+
+This will have the following net results:
+
+- **Debt Ceiling** remains at **0 DAI** after the transaction has been executed.
+- Dai generated from `PSM_PAX_A` increases by **754,598.72 DAI**.
+- USDP balance of `MCD_JOIN_PSM_PAX_A` increases by **754,598.72 USDP**.
+- USDP balance of `PSM_PAX_A_INPUT_CONDUIT_JAR` decreases to **0 USDP**.
+- Surplus Buffer increases by **754,598.72 DAI**.
+
+
+#### Yank BA Labs DAI and MKR Vest Streams
+
+If this executive proposal passes, following the recommendation of the Governance Facilitators, the BA Labs DAI and MKR vest streams will be canceled (yanked), namely:
+
+- yank BA Labs **DAI** stream **20**.
+- yank BA Labs **DAI** stream **21**.
+- yank BA Labs **MKR** stream **35**.
+
+### Trigger Spark Proxy Spell
+
+**Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/stability-scope-parameter-changes-9/23688/2), [Governance Poll](https://vote.makerdao.com/polling/QmQC1UXZ)\
+**Proposal:** [Forum post "Stability Scope Parameter Changes #9"](https://forum.makerdao.com/t/stability-scope-parameter-changes-9/23688), [Forum post "[Feb 14, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/feb-14-2024-proposed-changes-to-sparklend-for-upcoming-spell/23684)
+
+If this executive proposal passes, the SparkLend Proxy Spell (Ethereum) at TBD will be executed, containing the following major items:
+- Increase the [wstETH](https://app.spark.fi/reserve-overview/?underlyingAsset=0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0&marketName=proto_spark_v3) [Supply Cap](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps) to **1,2M**.
```

Comment:
:yellow_circle: **Change requested**

1. Change to decimal point
2. Add unit
3. Specify "million"

```suggestion
- Increase the [wstETH](https://app.spark.fi/reserve-overview/?underlyingAsset=0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0&marketName=proto_spark_v3) [Supply Cap](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps) to **1.2 million wstETH**.
```